hivemind/hivemind_cli/run_server.py

```diff
@@ -32,6 +31,11 @@ def main():
                         help="expert type from test_utils.layers, e.g. 'ffn', 'transformer', 'det_dropout' or 'nop'")
     parser.add_argument('--hidden_dim', type=int, default=1024, required=False, help='main dimension for expert_cls')
 
+    parser.add_argument('--host_maddrs', type=list, nargs='+', default=['/ip4/0.0.0.0/tcp/0'], required=False,
+                        help='Multiaddrs to listen for external connections from other p2p instances; default: all IPv4 and TCP: /ip4/0.0.0.0/tcp/0')
+    parser.add_argument('--announce_maddrs', type=list, nargs='+', default=None, required=False,
+                        help='Visible multiaddrs the host announces for external connections from other p2p instances')
+
```

Comment:
Now let's ensure that these args are passed into `hivemind.DHT` inside `hivemind.Server`. Probably, we need to pass them along with `initial_peers` [here](https://github.com/learning-at-home/hivemind/blob/master/hivemind/moe/server/server.py#L158).

Reply:
We are already passing them. `Server` starts as `Server.create(**args, ........)`. I added `**kwargs` to the `Server.create` which are further passed to the `DHT`, which passes it into `P2P`.

Maybe it is not the best way, but I saw how this arguments are passed from `DHT` into `P2P` (it is done through `**kwargs`) and done the same.

Reply:
Is it ok to pass this arguments through `**kwargs`?

Reply:
Oops, sorry, I looked at the wrong branch back then :) It's all good, I'm resolving this!